docs/targets/sendgrid.md

```diff
@@ -1,159 +1,56 @@
-
-2. Next move on to the '200-target.yaml' file in the same folder. Here was can imput some *optional* default values. If you do not require default values and want to pass these values dynamically via parameters in the CE body, you may omit some or all of these by deleteing the lines and leaving only the 'apiKey' portion.
-
-    * A list of (optional) supported spec options: defaultFromEmail, defaultToEmail, defaultToName, defaultFromName, defaultMessage
-
-3. Last file in the folder is ' 300-trigger.yaml' Here you can make modifications or leave it as is to subscribe to the default broker in the namespace you deploy. 
-
-
-### Status
-
-* The SendgridTarget requires only one Secret, the APIKey, to be provided. 
-
-* Once its presences is confirmed, the Sendgrid Target Controller will create a Knative Service for the Adapter.  
-
-* A Status summary is added to the SendgridTarget object informing of the all conditions that the target needs.
-
-* When ready the `status.address.url` will point to the internal point where Cloud Events should be sent.
+## Prerequisites
 
-### Sendgrid Target as an event Sink
+* Register a [SendGrid account][sgSU]
+* Retrieve a [SendGrid API token][api]
 
-#### Once deployed, Sendgrid Target Adapters are addressable. This means you can use it as a Sink for Knative components! Below you can find an example. 
+## Deploying an Instance of the Target
 
-* The included example trigger ['300-trigger.yaml'](https://github.com/triggermesh/knative-targets/blob/Sendgrid-merge-fix/sendgrid/samples/300-trigger.yaml) Is as follows. 
+Open the Bridge creation screen and add a target of type `SendGrid`.
 
-```yaml
-apiVersion: eventing.knative.dev/v1beta1
-kind: Trigger
-metadata:
-  name: sendgrid-sample-trigger
-spec:
-  broker: default
-  subscriber:
-    ref:
-      apiVersion: targets.triggermesh.io/v1alpha1
-      kind: SendgridTarget
-      name: triggermesh-email
-```
+![Adding a SendGrid target](../images/sendgrid-target/create-bridge-1.png)
 
-* Once this, along with the other aformentioned requirements, have been deployed. It is now possible to implement something like the [awskinesis-sinkbinding.yaml](https://github.com/triggermesh/aws-event-sources/blob/master/config/samples/awskinesis-sinkbinding.yaml) example from the [aws-event-sources](https://github.com/triggermesh/aws-event-sources).
+In the Target creation form, provide a name for the event Target, and add the following information:
 
-* By replacing the section starting at line [#18](https://github.com/triggermesh/aws-event-sources/blob/93df64d5b298cc73acfb93f835646cc1d06429c4/config/samples/awskinesis-sinkbinding.yaml#L18) with:
+* **Default sender name**: Assign a default sender name. (optional)
+* **Default sender email**: Assign a default sender email. (optional)
+* **Default recipient name**:  Assign a default recipient's  name. (optional)
+* **Default recipient email**: Assign a default recipient's email. (optional)
```

Comment:
_Assign_... to what? I have never used that target and am puzzled after reading this. From a user perspective it would be better for formulate these as

> Sender name to assign to XYZ if the received event does not already contain that information.

or something else along those lines, as long as it tells what the field is used for.